rel/overlay/etc/default.ini

```diff
@@ -1026,3 +1026,26 @@ url = {{nouveau_url}}
 ; are too many design documents, that may generate a lot of logs. The default
 ; is to aggregate reports per database.
 ;ddoc_report = false
+
+[couch_auth_lockout]
+; CouchDB can temporarily lock out IP addresses that repeatedly fail authentication
+; mode can be set to one of three recognised values;
+; off - CouchDB does not track repeated authentication failures
+; warn - CouchDB will log a warning if repeated authentication failures occur
+; enforce - CouchDB will reject requests with a 403 status code if repeated
+; authentication failures occur
+;mode = off
+
+; The number of authentication failures above which lockout occurs.
+;threshold = 5
+
+; to control memory usage CouchDB will only track authentication failure count
+; for this many username and IP address pairs.
+; note: changing this setting requires a couchdb restart.
+;max_objects = 10000
+
+; The maximum time, in milliseconds, that CouchDB will track repeated authentication
+; failures. The account is automatically unlocked at the end of this time, starting
+; from the _first_ authentication failure.
+; note: changing this setting requires a couchdb restart.
+;max_lifetime = 300000
```

Comment:
Add a newline to avoid having a red warning marker, and mess with some users editors where it would auto-append it on save later when we update the file again.